src/components/admin/Tables/PendingTable.js

```diff
@@ -56,13 +71,24 @@ export default function PendingTable() {
             </TableRow>
           </TableHead>
           <TableBody>
-            {computeRows(page, rows, rowsPerPage).map((row) => (
-              <TableRow key={row.id}>
-                <TableCell>{row.name}</TableCell>
-                <TableCell>{row.email}</TableCell>
-                <TableCell><MoreHorizIcon /></TableCell>
-              </TableRow>
-            ))}
+            {computeRows(page, rows, rowsPerPage)
+                .map((row: any): React.Node => (
```

Comment:
is it really necessary to annotate all arrow functions? Kinda defeats their purpose.

Reply:
maybe it's just the way I've configured the linter, but it throws errors otherwise. I can turn off that rule.

Reply:
I would do so, as it doesn't provide anything in my opinion